utils/moreutils/patches/001_disable-manuals.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
```

Comment:
@stintel 
That is incorrect, we have several packages where docs (and other parts) are disabled using patching to reduce build time. Most of the time that "feature" has been requested.